packages/gasket-plugin-docs/lib/index.d.ts

```diff
@@ -8,21 +9,80 @@ declare module '@gasket/engine' {
     }
   }
 
-  // TODO: correctly document what the full payload is...
-  type DocsConfigSet = {
-    docsRoot: string
-  };
+  export interface DocsSetupModulesConfig {
```

Comment:
Is it intentional to export these from `@gasket/engine`? If you export them from the top level instead would that make the importable from `@gasket/plugin-docs`?

Reply:
From my side there's no real intent. I was following existing design patterns of the file(s). I'm not a TypeScript expert by any means so I'll lean to your direction. @jpage-godaddy